mindmeld/components/_config.py

```diff
@@ -1102,6 +1104,12 @@ def get_tokenizer_config(app_path=None, exclude_from_norm=None):
         tokenizer_config = getattr(
             _get_config_module(app_path), "TOKENIZER_CONFIG", DEFAULT_TOKENIZER_CONFIG
         )
+        if not tokenizer_config.get("allowed_patterns"):
+            # If allowed_patterns are not provided, use default
+            tokenizer_config["allowed_patterns"] = []
+            tokenizer_config["default_allowed_patterns"] = _get_default_regex(
+                exclude_from_norm
+            )
```

Comment:
Good point. In this PR, `exclude_from_norm` should continue to work as it did before. In `_init_regex(self)` the `exclude_from_norm` symbols are added to the compiled regex.

`to_exclude = CURRENCY_SYMBOLS + "".join(self.exclude_from_norm)`
  
This now leaves the remaining normalization methods "ASCII Folding", "NFC", "NFD", "NFKC", "NFKD". Since these all normalize at the character level, I have added the following code which skips normalization if the character is in exclude_from_norm:

```
        final_norm_token_text = []
        for c in regex_norm_token_text:
            c = c if c in self.exclude_from_norm else self._normalizer.normalize(c)
            final_norm_token_text.append(c) 
```

This code is temporary since in Phase 2 it will be the user's responsibility to ensure that their Regex rules do not normalize a particular character. Although, this would work for the RegexNormalizer. I might be able to alter the pipeline to normalize around exclude_from_norm characters, for example:

"I got 80% on the test"
normalize("I got 80") +"%" normalize("on the test")